scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -137,15 +139,19 @@ def __init__(self, **kwargs):
         self.deploy_tpl_path = None
         self.forseti_conf_path = None
 
+        # forseti_conf.yaml.in properties
         self.skip_email = False
-        self.sendgrid_api_key = '""'
-        self.notification_sender_email = '""'
-        self.notification_recipient_email = '""'
+        self.sendgrid_api_key = kwargs.get('sendgrid_api_key')
```

Comment:
For unattended install mode, are there any checks or validations that certain flags have to be set?   I think that is really important to have.